en/docs/developer-guides/workspace-management/create-integration.md

```diff
@@ -0,0 +1,45 @@
+# Create an Integration
+
+## Creating an Integration Without Workspace
+When you need to build a simple, standalone integration, you can create it directly without setting up a workspace. This streamlined approach is ideal for independent services.
+
+**Steps:**
+
+1\. Launch WSO2 Integrator: BI and navigate to the integrations view.
+
+2\. Click the **Create New Integration** button.
+
+3\. Fill in **Create Your Integration** details and click **Create Integration**:
+
+- Provide the integration name
+- Provide a package name 
+- Select the path 
+- Provide the organization name and version in **“Optional Configurations”** (Optional)
+
+Now you will see the BI home page.
+
+<a href="{{base_path}}/assets/img/developer-guides/workspace/create-integration-hq.gif"><img src="../../../assets/img/developer-guides/workspace/create-integration-hq.gif" alt="Create Integration Without Workspace" width="80%"></a>
```

Comment:
Can we have proper names from a real world sample than using sample 1 etc. Also when opening the files to select the path, try to use a clean location without other jargons.